api/defaults.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package api
+
+import "time"
+
+const (
+	// DefaultDialTimeout is a default TCP dial timeout we set for our
+	// connection attempts
+	DefaultDialTimeout = 30 * time.Second
```

Comment:
Are all of these new constants, or have they moved from elsewhere? I couldn't find if they've been removed from somewhere else, so just making sure we don't have duplicates now?

Reply:
They are duplicated in `/lib/defaults/defaults.go`, I'll import them from `/api` there.

Reply:
Actually this causes an import cycle: 

```package github.com/gravitational/teleport/api
imports github.com/gravitational/teleport/api/proto
imports github.com/gravitational/teleport/lib/events
imports github.com/gravitational/teleport/lib/defaults
imports github.com/gravitational/teleport/api: import cycle not allowed```

We are going to break the dependency of api on events, so I'll remove the new `defaults.go` until then.